build/build-gh-pages.js

```diff
@@ -1,5 +1,30 @@
 const fs = require("fs").promises;
 
+
+// This object exist only to format the HTML as designed
+// type: warning-type as stored as filename in design/svg-folder
+// title: title that will be desplayed in html
+// Order is based on this site: https://hjelp.yr.no/hc/no/articles/360014052634
+//
+// "type: wind" will look for files named "icon-warning-wind-${yellow/orange/red}.svg"
+const warningTypes = [
+  {type: 'wind', title: 'Gust/Wind'},
+  {type: 'rain', title: 'Rain'},
+  {type: 'rainflood', title: 'Rain Flood'},
+  {type: 'lightning', title: 'Lightning'},
+  {type: 'snow', title: 'Snow'},
+  {type: 'snowstorm', title: 'Snow Storm'},
+  {type: 'icing', title: 'Ice'},
```

Comment:
Adding a note here since I can't comment on the SVG file itself. Should we no longer have an icon for the event type `ice`?

In `src/components/WarningIcon/WarningIcon.tsx` we show `icon-warning-ice` when event type is `Ice` and `icon-warning-generic` when event type is `Icing`. We would need to update this component in that case, but is this really something we should do? We need to talk with the designers to find out.